InsertValue.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Agumon', DATE '2020-02-03', 0, true, 10.23);
```

Comment:
- [ ] Kindly in this project we are required to copy all the SQL used in into the `data.sql` file in the generated repository, not `InsertValue` 😀😀

Reply:
Hi @TuyishimireEric,
thanks for your good and valuable feedback.
as you mentioned I change the Insert Value to data. SQL.